scripts/modules/validation/sanitizers.js

```
@@ -0,0 +1,667 @@
+		global.projectName = global.projectName.trim();
+	}
+
+	// Normalize default priority
+	if (global.defaultPriority) {
+		global.defaultPriority = global.defaultPriority.toLowerCase().trim();
+		if (!['high', 'medium', 'low'].includes(global.defaultPriority)) {
+			log(
+				'warn',
+				`Invalid defaultPriority: ${global.defaultPriority}, using default`
+			);
+			delete global.defaultPriority;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)